File: Native/Test/Fabric/binding.js
F = require('Fabric').createClient();

b = F.DG.createBinding();
console.log(JSON.stringify(b.getOperator()));
console.log(JSON.stringify(b.getParameterLayout()));

o = F.DG.createOperator("op");
b.setOperator(o);
console.log(JSON.stringify(b.getOperator().getName()));

b.setParameterLayout(["self.foo"]);
console.log(JSON.stringify(b.getParameterLayout()));

F.flush();

File: Native/Test/Python/binding.py
#
#  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
#

import json
import fabric
F = fabric.createClient()

b = F.DG.createBinding()
print(fabric.stringify(b.getOperator()))
print(fabric.stringify(b.getParameterLayout()))

o = F.DG.createOperator("op")
b.setOperator(o)
print(fabric.stringify(b.getOperator().getName()))

b.setParameterLayout(["self.foo"])
print(fabric.stringify(b.getParameterLayout()))

F.close()

